oem/ibm/configurations/bios/enum_attrs.json

```diff
@@ -573,6 +573,28 @@
             "default_values": ["MTU1500"],
             "helpText": "Specifies the maximum frame size (maximum transmission unit, MTU). The value is in terms of bytes per packet size.",
             "displayName": "Maximum Frame Size"
+        },
+        {
+            "attribute_name": "pvm_default_os_type",
```

Comment:
We already have a existing pvm_default_os_type why you want to create a new one here ?

Reply:
this commit is only changing - possibleValues, defaultValues, helpText, displayName for pvm_default_os_type. Why is code from line 576 to 578 being added as a part of your commit

Reply:
removed the pvm_default_os_type as it is already defined

Reply:
In the initial commit change you changes some values for pvm_default_os_type, why are we removing all of it? Is it not required ? As per the attribute doc some values need change, shouldn't this be part of this commit?